packs/rss/sensors/rss_sensor.yaml

```diff
@@ -0,0 +1,28 @@
+---
+  class_name: "RSSSensor"
+  entry_point: "rss_sensor.py"
+  description: "Sensor which monitors RSS / Atom feed for new entries"
+  poll_interval: 15
+  trigger_types:
+    -
+      name: "feed"
```

Comment:
Currently there is a another issue with triggers with parameters - we call `add_trigger` on sensor instance for every trigger which belongs to that sensor.

In this case this means we also call it for `rss.entry` triggers which makes little sense (this is an outbound trigger which is dispatched by a sensor and not something a sensor is interested in).

I propose to make a change to only call trigger handler methods for triggers with parameters.

Feedback?